persistence/src/commonMain/db_user/com/wire/kalium/persistence/Conversations.sq

```diff
@@ -240,6 +240,7 @@ WHERE
     OR (type IS 'ONE_ON_ONE' AND userDeleted = 1) -- show deleted 1:1 convos, to maintain prev, logic
     )
     AND (protocol IS 'PROTEUS' OR (protocol IS 'MLS' AND mls_group_state IS 'ESTABLISHED'))
+    AND (archived = ? OR archived IS NULL)
```

Comment:
Is it possible that archived can be null?

Reply:
I'm not sure, I thought that it could be null values for versions prior to archiving, but I assume that after the migration they will be converted to the default value, i.e 0

Reply:
pushed